### DISCUSSION TITLE
My placeholders doesn't work

### DISCUSSION THREAD

**Discussion by Ignius14** [PlaceholderAPI/PlaceholderAPI#833] [(2) Support]
Hi, I was shearching like whole internet for solution in placeholders.
SuperiorSkyblock2 (https://www.spigotmc.org/resources/%E2%9A%A1%EF%B8%8F-superiorskyblock2-%E2%9A%A1%EF%B8%8F-the-best-core-on-market-%E2%9A%A1%EF%B8%8F-1-18-support.87411/)
And vault economy placeholders doens't work.

I don't get any errors.

I have all extensions.
And all plugins are latest version.

I need help.

**Comment by Andre601** [comment]
Please provide information such as the output from `/papi dump`, your latest log files and what you tried so far.

Just saying "it doesn't work" and that you use "the latest versions" of everything doesn't help us here. We need proper information to help.

**Reply by Ignius14** [reply]
https://paste.helpch.at/ganuwadeyu

**Reply by Ignius14** [reply]
This is /papi dump outcome

**Reply by Andre601** [reply]
There is no superiorskyblock extension loaded, so you either didn't download it, or SS2 doesn't provide any.

Also, what vault stuff doesn't work actually?

**Reply by Ignius14** [reply]
%eco_vault_balance%, {island-worth} and more. So i don't know what to do.
Btw placeholderapi doesn't provide any SS2 extensions so yeah
